QModManager/API/SMLHelper/Patchers/PDAPatcher.cs

```diff
@@ -24,9 +24,7 @@ internal static void Patch(HarmonyInstance harmony)
         private static void InitializePostfix()
         {
             BlueprintToFragment.Clear();
-
-            // Direct access to private fields made possible by https://github.com/CabbageCrow/AssemblyPublicizer/
```

Comment:
We should either keep these comments or have a link to the AssemblyPublicizer project somewhere visible in the repo.  
Anyone looking to contribute to the SMLHelper APIs of QModManager may very well get confused when they see us directly accessing various private elements without Reflection.

Reply:
Documentation will be the last thing for 3.0